crates/router/src/connector/iatapay.rs

```diff
@@ -90,6 +90,10 @@ impl ConnectorCommon for Iatapay {
         "application/json"
     }
 
+    fn get_currency_unit(&self) -> api::CurrencyUnit {
+        api::CurrencyUnit::Minor
```

Comment:
Hey @SoumyoPurkayastha This will be `base` as Iatapay accepts `f64`, which would be something like $1.2 


```suggestion
        api::CurrencyUnit::Base
```